### ISSUE TITLE
[MU4 Issue] Accessibility: NVDA does not read the fields changing names in Instruments panel

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#15743]
**Describe the bug**
When changing staff names via the Instrument panel, NVDA does not read the names

**To Reproduce**
1. Start NVDA
2. Open a score
3. Go to Instruments panel
4. Click gear icon to access staff names
5. Hear error: NVDA reads only "MuseScore 4 Windows", and does not read as you navigate between the full & abbreviated names

**Expected behavior**
NVDA should read the names of the filed and their contents

**Platform information**
 - OS: Windows

**Additional context**
It works as expected with JAWS, and also with Orca on Linux.

**Comment by jmwmusic** [comment]
Is this a duplicate of #25404
